src/Html.php

```diff
@@ -5612,7 +5609,8 @@ function(index, file) {
                         $('#progress{$p['rand']}').show();
                         $('#progress{$p['rand']} .uploadbar')
                            .text(file.error)
-                           .css('width', '100%');
+                           .css('width', '100%')
+                           .show();
```

Comment:
when a user uploads an image with acceptable size (< 2MB) this element shows a success message then fades out and display is disabled with style.

This line ensures it becomes visible to show the error when a 2nd image (too big) is dropped in the textarea)